SimCore/include/SimCore/XsecBiasingOperator.h

```diff
@@ -177,7 +177,7 @@ class XsecBiasingOperator : public G4VBiasingOperator {
     return nullptr;
   }
   /// Enable logging
-  enableLogging("EcalProcessFilter")
+  enableLogging("XsecBiasingOperator")
```

Comment:
Instead of hard-coding the channel name, I would like the channel name to align with the derived biasing operator's name. We can do this by avoiding the `enableLogging` macro.

This is done by event processors among other factory-created objects.

https://github.com/LDMX-Software/ldmx-sw/blob/446e5c3345e00ee7aff45e249d6502d2e69b861f/Framework/include/Framework/EventProcessor.h#L221-L222

https://github.com/LDMX-Software/ldmx-sw/blob/446e5c3345e00ee7aff45e249d6502d2e69b861f/Framework/src/Framework/EventProcessor.cxx#L13

Reply:
So your suggestion is to simply
```suggestion
```

Reply:
You would remove `enableLogging` and then add the declaration `logging::logger theLog_;` in the header and definition `theLog_{logging::makeLogger(name)}` in the constructor's definition in the source.

Reply:
Done in https://github.com/LDMX-Software/ldmx-sw/pull/1630/commits/772493fcbd35d12c466a8c4d7be7066af0fa4d58